e2e/testdata/fn-render/kubeval-failure/Kptfile

```diff
@@ -3,7 +3,7 @@ kind: Kptfile
 metadata:
   name: app
 pipeline:
-  mutators:
-  - image: gcr.io/kpt-fn/kubeval:v0.1
+  validators:
+  - image: gcr.io/kpt-fn/kubeval:unstable
```

Comment:
Why are we moving to `unstable` here ?

Reply:
We improve the error message in GoogleContainerTools/kpt-functions-catalog#297. It has not been released yet.
We use it because its error is more readable and it doesn't have empty newline in the message.